init.sql

```diff
@@ -257,6 +315,22 @@ ALTER TABLE ONLY public.transcripts
     ADD CONSTRAINT transcripts_pkey PRIMARY KEY (id);
 
 
+--
+-- Name: transcripts transcripts_transcriptHash_key; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.transcripts
+    ADD CONSTRAINT "transcripts_transcriptHash_key" UNIQUE ("transcriptHash");
+
+
+--
+-- Name: tutorials tutorials_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.tutorials
+    ADD CONSTRAINT tutorials_pkey PRIMARY KEY (id);
+
```

Comment:
what are tutorials?

Reply:
this was data used to test during initial set up. i have since removed it